packages/functional-tests/tests/oauth/loginHint.spec.ts

```diff
@@ -1,97 +1,103 @@
-    await relier.goto(`email=${invalidEmail}`);
-    await relier.clickEmailFirst();
-    const error = await login.getTooltipError();
-    expect(error).toContain('Valid email required');
-  });
-
-  test('login_hint specified by relier, not registered', async ({
-    pages: { login, relier },
-  }) => {
-    const email = login.createEmail();
-    await relier.goto(`login_hint=${email}`);
-    await relier.clickEmailFirst();
-
-    // Email is prefilled
-    await expect(await login.getPrefilledEmail()).toEqual(email);
-    expect(await login.signUpPasswordHeader()).toEqual(true);
-
-    await login.useDifferentAccountLink();
-
-    // Email first page has email input prefilled
-    await expect(await login.getEmailInput()).toEqual(email);
-  });
-
-  test('email specified by relier, registered', async ({
-    credentials,
-    pages: { login, relier },
-  }) => {
-    await relier.goto(`email=${credentials.email}`);
-    await relier.clickEmailFirst();
-
-    // Email is prefilled
-    await expect(await login.getPrefilledEmail()).toEqual(credentials.email);
-    expect(await login.signInPasswordHeader()).toEqual(true);
-
-    await login.useDifferentAccountLink();
-
-    // Email first page has email input prefilled
-    await expect(await login.getEmailInput()).toEqual(credentials.email);
-  });
-
-  test('login_hint specified by relier, registered', async ({
-    credentials,
-    pages: { login, relier },
-  }) => {
-    await relier.goto(`login_hint=${credentials.email}`);
-    await relier.clickEmailFirst();
-
-    // Email is prefilled
-    await expect(await login.getPrefilledEmail()).toEqual(credentials.email);
-    expect(await login.signInPasswordHeader()).toEqual(true);
-
-    await login.useDifferentAccountLink();
-
-    // Email first page has email input prefilled
-    await expect(await login.getEmailInput()).toEqual(credentials.email);
-  });
-
-  test('cached credentials, login_hint specified by relier', async ({
-    target,
-    pages: { login, relier },
-  }) => {
-    const email = login.createEmail();
-    await target.createAccount(email, PASSWORD);
-
-    const loginHintEmail = login.createEmail();
-    await target.createAccount(loginHintEmail, PASSWORD);
-
-    // Create a cached login
-    await relier.goto();
-    await relier.clickEmailFirst();
-    await login.login(email, PASSWORD);
-    expect(await relier.isLoggedIn()).toBe(true);
-    await relier.signOut();
-
-    // login_hint takes precedence over the signed-in user
-    await relier.goto(`login_hint=${loginHintEmail}`);
-    await relier.clickEmailFirst();
-
-    // Email is prefilled
-    await expect(await login.getPrefilledEmail()).toEqual(loginHintEmail);
-    expect(await login.signInPasswordHeader()).toEqual(true);
-
-    await login.useDifferentAccountLink();
-
-    // Email first page has email input prefilled
-    await expect(await login.getEmailInput()).toEqual(loginHintEmail);
+test.describe('severity-2 #smoke', () => {
```

Comment:
A bit on the fence about a nested `describe` block that only contains one `describe` block. It is probably fine if it helps organize things.

Reply:
Not that it's a vote, but I kinda dig it because (a) it's consistent w/ other files above, (b) if we add more tests below within the parent `test.describe('severity-2 #smoke', () => ...)` block, they'd get their severity automatically set (versus if we added the tags to the `test.describe()` below on L10.

Reply:
I just kept it that way to be consistent with all our test files.